test/lib/mocked-dockerode.ts

```diff
@@ -0,0 +1,57 @@
+import * as dockerode from 'dockerode';
+
+export interface TestData {
+	networks: Dictionary<any>;
+	images: Dictionary<any>;
+	containers: Dictionary<any>;
```

Comment:
I'd just remove this for now, as you don't implement any way to retrieve that data